lifecycle-operator/chart/README.md

```diff
@@ -20,6 +20,7 @@ and application health checks
 | `global.commonLabels`                                   | Common labels to add to all Keptn resources. Evaluated as a template                                                                                          | `{}`                |
 | `global.commonAnnotations`                              | Common annotations to add to all Keptn resources. Evaluated as a template                                                                                     | `{}`                |
 | `global.caInjectionAnnotations`                         | CA injection annotations for cert-manager.io configuration                                                                                                    | `{}`                |
+| `global.isOpenshift`                                    | Handles specific settings for Openshift                                                                                                                       | `false`             |
```

Comment:
```suggestion
| `global.isOpenshift`                                    | Settings for Openshift                                                                                                                       | `false`             |
```
Note that the other items in the table do not include a verb so let's keep it parallel

Reply:
Since I've now changed the parameter to `global.openShift.enabled` I'm dubious about what the description should be.

On `global.certManagerEnabled` the description is "Enable this value to install Keptn Certificate Manager" and therefor I've now written "Enable this value to install on Openshift". What do you think?